### PR TITLE
fix(using bignumber): using BigNumber instead of number

### PR DESCRIPTION
**Pull request by CalicoNino**
A part of #468 fix

### CODE REVIEW

File: packages/nibijs/src/test/faucet.test.ts [hunk 1]
```diff
@@ -45,8 +45,8 @@ test("faucet utility works", async () => {
   const balancesEnd = newCoinMapFromCoins(
     await signingClient.getAllBalances(toAddr)
   )
-  expect(balancesEnd.unusd - balancesStart.unusd).toEqual(100 * 1e6)
-  expect(balancesEnd.unibi - balancesStart.unibi).toEqual(11 * 1e6)
+  expect(balancesEnd.unusd.minus(balancesStart.unusd)).toEqual(100 * 1e6)
+  expect(balancesEnd.unibi.minus(balancesStart.unibi)).toEqual(11 * 1e6)
```
Comment (cgilbe27) [inline packages/nibijs/src/test/faucet.test.ts]: BigNumber itself has assertions https://mikemcl.github.io/bignumber.js/#eq

**Comment by CalicoNino** [comment]
@cgilbe27 btw, the indexer unit test that is failing is also failing on the default branch

File: packages/nibijs/docs/classes/StableSwap.md [hunk 2]
```diff
@@ -178,7 +178,7 @@ y()
 Calculate x[j] if one makes x[i] = x
 
 Done by solving quadratic equation iteratively.
-x*1**2 + x1 * (sum' - (A*n**n - 1) * D / (A _ n**n)) = D ** (n+1)/(n \*\* (2 _ n) \_ prod' \* A)
+x*1\*\*2 + x1 * (sum' - (A*n\*\*n - 1) * D / (A _ n**n)) = D ** (n+1)/(n \*\* (2 _ n) \_ prod' \* A)
```
Comment (cgilbe27) [inline packages/nibijs/docs/classes/StableSwap.md]: Is this correct?

**Review by cgilbe27 — APPROVED** [review]

File: packages/nibijs/docs/classes/StableSwap.md (hunk 2 above)
Comment (CalicoNino) [reply packages/nibijs/docs/classes/StableSwap.md]: not sure, it was the linter, reverting it so reduce scope of PR and get it merged

**Pull request merged**